charts/mosip-file-server/values.yaml

```diff
@@ -52,7 +52,7 @@ service:
 image:
   registry: docker.io
   repository: mosipqa/mosip-file-server
-  tag: 1.2.0.1
+  tag: develop
```

Comment:
It should be release-1.2.0.1 na?

Reply:
release-1.2.0.1 branch is not present , so syed suggested to update in 1.2.0.1 .